src/annotate.spec.ts

```diff
@@ -97,12 +99,9 @@ test('Testing Zooma annotate service', (t: Test) => {
 
     const result = await _annotate.default({ payload: { field, term } })
       .then(function(data: any) {
-        //console.log('got data', data);
         return data;
       }).catch((err: any) => {
-        console.error(err);
-        console.log('annotate error');
-        // API call failed...
+        st.ok(err, 'There is an error');
```

Comment:
@Istar-Eldritch I wanted to check that an error is thrown, but this is the only way I could get it to work. Is there a better way?

Reply:
Yep, this is correct. :)

Reply:
OK thanks.
Let me know if there's anything else I need for this PR. I think it should be complete now.

Reply:
Also, if you are able to set up/import the environment variables for metis on CircleCI that would be great.

Reply:
Sure. Sorry about the CI thing, we need to check why you don't have permissions on it.

Reply:
I think you need to be a github admin

Reply:
On the import variables button for me it says:
"To use this function you must be an organisation admin for your VCS-provider"

Reply:
That would make sense. I just added the env vars. I would like to review this properly one more time wefore giving green light ;)